### ISSUE TITLE
Automated Testing Build Error (9/20/2016 11:33 AM)

### ISSUE THREAD

**Issue by phet-steele** [phetsims/build-an-atom#135]
```
Running "eslint:allFiles" (eslint) task

/home/mendeleev/git/shred/js/model/ParticleAtom.js
   44:9   error  Unexpected alias 'thisAtom' for 'this'  consistent-this
  164:11  error  Unexpected alias 'thisAtom' for 'this'  consistent-this
  309:11  error  Unexpected alias 'thisAtom' for 'this'  consistent-this

✖ 3 problems (3 errors, 0 warnings)
```

**Comment by samreid** [comment]
Fixed in 531be01f6bd20a15d8196380bfe97f2409c87a02 @phet-steele can you verify?

**Comment by phet-steele** [comment]
👍
